src/main/java/co/inventorsoft/springresttask/controller/dto/EmailDto.java

```diff
@@ -0,0 +1,20 @@
+package co.inventorsoft.springresttask.controller.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.sql.Timestamp;
+
+@Data
+@Builder
+public class EmailDto {
+
+    private int id;
+    private String sender;
+    private String receiver;
+    private String theme;
+    private String content;
+    private boolean sent;
+    private Timestamp date;
```

Comment:
It's better to avoid SQL dates, have a look at Java Time API. It also works well with Jackson